CRM/Upgrade/Incremental/php/FourSeven.php

```diff
@@ -508,6 +512,19 @@ public static function convertBackendToSettings($domainId, $config_backend) {
   }
 
   /**
+   * Update Invoice number for all completed contribution.
+   *
+   * @param \CRM_Queue_TaskContext $ctx
+   *
+   * @return bool
+   */
+  public static function updateContributionInvoiceNumber(CRM_Queue_TaskContext $ctx, $invoicePrefix) {
+    $sql = "UPDATE `civicrm_contribution` SET `invoice_number` = CONCAT(%1, `id`)";
```

Comment:
Setting this to WIP since upgrade code need to handle
1. Set `invoice_number` for `invoice_number` IS NOT NULL
2. Run Update query in batches.